DQMOffline/L1Trigger/interface/L1TMuonOffline.h

```diff
@@ -110,10 +110,10 @@ class MuonGmtPair {
 // DQM class declaration
 //
 
-class L1TEfficiencyMuons_Offline : public DQMEDAnalyzer {
+class L1TMuonOffline : public DQMEDAnalyzer {
```

Comment:
hi @thomreis - while we are moving the name of this around- it ought to include DQM somewhere in its name. Thanks